### DISCUSSION TITLE
App per mobili

### DISCUSSION THREAD

**Discussion by francyoo17** [iisgiua/giuaschool#697] [Richieste di aiuto] [answered]
Salve esiste app per telefono di giuaschool

**Comment by trinko** [comment, marked as answer]
Vedi [App giuaReg 3.0 per l'utilizzo del registro elettronico giuaSchool](https://github.com/iisgiua/giuaschool/discussions/444)

**Comment by ghost** [comment]
@francyoo17 Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
